akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala

```diff
@@ -225,6 +227,7 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter {
       checkLatch(breaker.halfOpenLatch)
 
       // transit to open again
+      breaker.openLatch.reset
```

Comment:
The failing test was on the assert below:

```scala
 (shortRemainingDuration < longRemainingDuration) should ===(true)
```

That is testing the exponential backoff and the explanation would be that we checked too early, it had not performed the exponential increase yet.